kalitectl.py

```diff
@@ -372,6 +373,14 @@ def start(debug=False, args=[], skip_job_scheduler=False):
     # TODO: What does not the production=true actually do and how can we
     # control the log level and which log files to write to
 
+    # Deal with command-line options that affect the django settings module
```

Comment:
It would be nicer if you used the docopt interface for this, isn't that possible? Keeping a consistent syntax would help and would put the entries in usage specification as well.